docs/architecture/hotspot-makers/hotspot-makers.mdx

```diff
@@ -214,8 +214,7 @@ The FreedomFI Hotspot can work as an IOT Hotspot, a MOBILE CBRS Hotspot or opera
 
 - [Website](https://www.kerlink.com) Sales via distributors
 - Support: support@kerlink.com
-- [iOS App](https://apps.apple.com/fr/app/kerlink-helium-hotspot/id1613698129) -
-  [Android App](https://play.google.com/store/apps/details?id=com.kerlink.heliumhotspot&hl=fr&gl=US)
+- No app. Onboarding process at [Kerlink Solana Onboarding](https://wikikerlink.fr/helium/doku.php?id=wiki:solana:onboarding)
```

Comment:
```suggestion
- [Web App](https://wikikerlink.fr/helium/doku.php?id=wiki:solana:onboarding)
```
I suggest keeping consistent formatting with the other sections.

Additionally, the link is to a help article. Would it be better to link directly to the onboarding tool? https://helium-onboarding.kerlink.com